src/pages/MyKiva/MyKivaPageContent.vue

```diff
@@ -300,38 +297,32 @@ const loadMoreUpdates = () => {
 const showSingleArray = computed(() => loans.value.length === 1 && loanUpdates.value.length === 1);
 
 const fetchMyKivaData = () => {
-	return apollo.query({ query: myKivaQuery })
-		.then(result => {
-			userInfo.value = result.data?.my ?? {};
-			lender.value = result.data?.my?.lender ?? null;
-			lender.value = {
-				...lender.value,
-				public: userInfo.value?.userAccount?.public ?? false,
-				inviterName: userInfo.value?.userAccount?.inviterName ?? null,
-			};
-			loans.value = result.data?.my?.loans?.values ?? [];
-			totalLoans.value = result.data?.my?.loans?.totalCount ?? 0;
-			if (loans.value.length > 0) {
-				showLoanFootnote.value = loans.value.some(l => hasLoanFunFactFootnote(l));
-			}
-		}).catch(e => {
-			logReadQueryError(e, 'MyKivaPage myKivaQuery');
-		});
+	const result = apollo.readQuery({ query: myKivaQuery });
```

Comment:
We probably want a try catch around the `readQuery` here and in `fetchContentfulHeroData`